okhttp/build.gradle.kts

```diff
@@ -16,6 +16,12 @@ dependencies {
     implementation(libs.kotlin.coroutines.core)
 
     api(project(":library"))
+
+    testImplementation(libs.assertj)
+    testImplementation(libs.okhttp.mockwebserver)
+    testImplementation(libs.kotlin.coroutines.test)
+    testImplementation(project(":extensions:google-java"))
+    testImplementation(project(":conformance:google-java"))
```

Comment:
I don't think we want the okhttp module to depend on the conformance module - perhaps we just need a simple Eliza codegen in okhttp for tests so we have a service we can use.

Reply:
Sure, that makes sense. Would that be set up similarly to how the `:examples:generated-google-java` and `:examples:generated-google-javalite` modules are configured?